### PR TITLE
Patron page fixes

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

n/a

#### Description

- added missing props to patron page
- changed pincode length to max/min pincode instead
- renamed some urls (better names now!)
- added a modal on pause reservations 
- made a figure more accessible

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

File: src/apps/patron-page/sections/PincodeSection.tsx [hunk 1]
```diff
@@ -11,7 +11,8 @@ const PincodeSection: FC<PincodeSectionProps> = ({ changePincode }) => {
   const t = useText();
   const config = useConfig();
 
-  const pincodeLength = parseInt(config("pincodeLengthConfig"), 10) || 4;
+  const pincodeLengthMin = parseInt(config("pincodeLengthMinConfig"), 10) || 4;
+  const pincodeLengthMax = parseInt(config("pincodeLengthMaxConfig"), 10) || 4;
```
Comment (spaceo) [inline src/apps/patron-page/sections/PincodeSection.tsx:15]: The length should be stored in config if it is not configurable by the cms

File: src/apps/patron-page/sections/ReservationDetailsSection.tsx [hunk 2]
```diff
@@ -50,29 +46,17 @@ const ReservationDetailsSection: FC<ReservationDetailsSectionProps> = ({
...
       <CheckBox
         className="mt-32 mb-16"
         id="show-reservation-pause-section"
-        onChecked={() =>
-          setReservationPauseSectionVisible(!reservationPauseSectionVisible)
-        }
+        onChecked={openPauseReservationModal}
         ariaLabel={t("patronPageOpenPauseReservationsSectionAriaText")}
-        selected={reservationPauseSectionVisible}
+        selected={false}
```
Comment (spaceo) [inline src/apps/patron-page/sections/ReservationDetailsSection.tsx:57]: Is this `false` on purpose?

**Review by spaceo — CHANGES_REQUESTED** [review]
Splendid. Just a few remarks.

File: src/apps/patron-page/sections/ReservationDetailsSection.tsx (hunk 2 above)
Comment (sinejespersen) [reply src/apps/patron-page/sections/ReservationDetailsSection.tsx:57]: Yes

Comment (spaceo) [reply src/apps/patron-page/sections/ReservationDetailsSection.tsx:57]: 👍

**Comment by spaceo** [comment]
LGTM 👍

**Review by spaceo — APPROVED** [review]

**Pull request merged**